app/src/main/java/com/github/sdp/mediato/data/Utils.java

```diff
@@ -0,0 +1,78 @@
+package com.github.sdp.mediato.data;
+
+import com.github.sdp.mediato.model.Location;
+import com.github.sdp.mediato.model.User;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.ValueEventListener;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * This is a Utils class for the database classes
+ */
+class Utils {
+
+    //-----------Constant definitions-------------
+
+    static final String USERS_PATH = "Users/";
+    static final String REVIEWS_PATH = "reviews/";
+    static final String USER_COLLECTIONS_PATH = "/collections/";
+    static final String LOCATION_PATH = "/location/";
+    static final String FOLLOWING_PATH = "/following/";
+    static final String FOLLOWERS_PATH = "/followers/";
+
+    static final String USER_PROFILE_PICS_PATH = "ProfilePics/";
+
+    static final int PROFILE_PIC_MAX_SIZE = 1024 * 1024; //1 Megabyte
+
+    static final int LOCATION_RADIUS = 100;
+
+    //---------------------Util methods-------------------------------------
+
+    /**
+     * Helper method that returns the database reference for a collection
+     *
+     * @param username       the username of the user concerned
+     * @param collectionName the name of the collection needed
+     * @return the database reference for the collection
+     */
+    static DatabaseReference getCollectionReference(String username, String collectionName) {
+        return CollectionsDatabase.database.getReference().child(USERS_PATH + username + USER_COLLECTIONS_PATH + collectionName);
+    }
+
+    /**
+     * Helper method for UserDatabase that returns the nearby users and completes the future
+     *
+     * @param future to track the state
+     * @param location center location of the current user
+     * @param username username of the current user
+     * @return a list of strings with the users' usernames
+     */
+    static List<String> findNearbyUsers(CompletableFuture<List<String>> future, Location location, String username) {

```

Comment:
Same thing here, I would add the search radius as a parameter of the function, and then pass it to `isInRadius()`.